### PR TITLE
WFLY-10009 Introduce a maximum-timeout management model attribute

### PR DESCRIPTION
**Pull request by zhfeng**
https://issues.jboss.org/browse/WFLY-10009
https://issues.jboss.org/browse/EAP7-981

### CODE REVIEW

File: transaction/WFLY-10009_EAP7-981_Introduce_Maximum_Timeout.adoc [hunk 1]
```diff
@@ -0,0 +1,62 @@
+= Title
```
Comment (jamezp) [inline transaction/WFLY-10009_EAP7-981_Introduce_Maximum_Timeout.adoc]: This should be changed to a meaningful title

**Comment by istraka** [comment]
Thank you for the changes @zhfeng. IMHO It is good to go.

**Comment by zhfeng** [comment]
@jamezp is this OK to merge ?

**Comment by jamezp** [comment]
@zhfeng This will be merged when the WildFly PR is merged.

**Comment by zhfeng** [comment]
The Wildfly PR https://github.com/wildfly/wildfly/pull/11440 is merged.

**Pull request merged**